fanctrl.py

```diff
@@ -119,11 +113,9 @@ def getCurrentStrategy(self):
         return self.configuration.getDischargingStrategy()
 
     def getBatteryChargingStatus(self):
-        with open(self.configuration.batteryChargingStatusPath(), "r") as fb:
-            currentBatteryStatus = fb.readline().rstrip("\n")
-            if currentBatteryStatus == "Discharging":
-                return False
-            return True
+        bashCommand = "ectool battery"
+        rawOut = subprocess.run(bashCommand, stdout=subprocess.PIPE, shell=True, text=True).stdout
```

Comment:
Curiously, the output of this command is always starting with an error on stderr (at least on my 13" laptop, even with latest ectool):
```
EC result 3 (INVALID_PARAM)
Battery info:
  OEM name:               NVT
  Model number:           Framewo
  Chemistry   :           LION
  Serial number:          00CF
  Design capacity:        3572 mAh
  Last full charge:       3042 mAh
  Design output voltage   15400 mV
  Cycle count             386
  Present voltage         17359 mV
  Present current         0 mA (discharging)
  Remaining capacity      3042 mAh
  Flags                   0x07 AC_PRESENT BATT_PRESENT DISCHARGING
  ```
  Do you also have it ? It kinda pollutes the service outputs for no apparent reason, so its stderr output should be discarded

Reply:
I do not at the moment (I am on a 16" model).
I will discard the stderr